### ISSUE TITLE
backdropcms.org home page redesign & related changes

### ISSUE THREAD

**Issue by jenlampton** [backdrop-ops/backdropcms.org#301]
TODO: 
- [x] Redo all layouts to remove double sidebar https://github.com/backdrop-ops/backdropcms.org/issues/350
- [x] Theme footer region https://github.com/backdrop-ops/backdropcms.org/issues/353
- [x] Theme responsive drop-down menu https://github.com/backdrop-ops/backdropcms.org/issues/352
- [x] Reorganize main menu links https://github.com/backdrop-ops/backdropcms.org/issues/313
- [x] Add missing header elements from design https://github.com/backdrop-ops/backdropcms.org/issues/351
- [x] Build out new home page elements https://github.com/backdrop-ops/backdropcms.org/issues/355
- [x] Build new "install backdrop" page https://github.com/backdrop-ops/backdropcms.org/issues/314
- [x] Complete "Key features" content for new home page https://github.com/backdrop-ops/backdropcms.org/issues/356

## Original issue: 

Now that backdrop has been out for almost 2 years, perhaps we should refocus our home page to do a better job of selling backdrop to those who are curious about it. What do we put on here, how should it be organized, and what should it look like? So many hard questions for developers!

I'm going to have a meeting with Jordan Ryan next week to discuss some of these questions and see if we can form a plan of attack.  I'd like to plan to launch the new home page around Backdrop's 2-year birthday, perhaps right after 1.6 is released. We'll see how overly optimistic that is next week :)

**Comment by mikemccaffrey** [comment]
Let's try to discuss this at badcamp either as part of the summit or afterwards.

**Comment by wesruv** [comment]
![borg-homepage](https://cloud.githubusercontent.com/assets/5607236/20200772/421b721a-a781-11e6-916c-ea3c732c86bd.png)

A few notes:
* Thinking the picture in the features section (Easy Editing) could be short ~1min videos
* Thinking the screen shots in the phone and the monitor in the hero could be a carousel, the 'Default' text could switch to the name of a case study, or the name of a contrib theme
* Want to make the hero a little more interesting, maybe some sort of texture or something cool in the flat blue area

**Comment by tomgrandy** [comment]
Excellent!  

The current site is very text-centric and this one is more visually appealing.  The addition of sites that use Backdrop and the short videos would go a long way in encouraging people to see it before they download and install (though the Pantheon route makes that so much easier).

Are the top-tier menu items the ones that will be used?  What options do we have for the menu?

Can we do anything that has multi-columns in the menu?  We use these a great deal to allow us to put more items in the menu by sub-category and also allow users to get to what they are looking for quickly.
<img width="1241" alt="screen shot 2016-11-10 at 8 32 49 pm" src="https://cloud.githubusercontent.com/assets/660545/20201321/0fc88b64-a785-11e6-9d85-f3e0a3c34ac7.png">

**Comment by wesruv** [comment]
I think we could work that out, kind of need to given how many items are in the Handbook section. The goal is to leverage the responsive menu system in Backdrop core as much as possible, but I think we can have our cake and eat good UX too.

Looking at it some more I'm thinking the features section could use a link to a features page that's has more features and maybe a little extra meat.

**Comment by tomgrandy** [comment]
Agree.

You get the main idea in the feature paragraph, a visual overview with the feature video, but a read more link to the meat is a good idea.

I don't think there would be that many tiers under the main menu, but this is what the above site menu looks like on mobile:

<img width="399" alt="screen shot 2016-11-10 at 8 45 12 pm" src="https://cloud.githubusercontent.com/assets/660545/20201518/bdc351a8-a786-11e6-9298-f1683aa5d118.png">

**Comment by klonos** [comment]
Loving this @wesruv 👍 ❤️

**Comment by jenlampton** [comment]
Love!

- -
Please excuse brevity and typos. Message typed with my thumbs from an Open
Source OS.

**Comment by wesruv** [comment]
Aha! Textures make some of the plainer parts of this feel a lot better.
![borg-homepage](https://cloud.githubusercontent.com/assets/5607236/20221905/3ceb1fcc-a801-11e6-86de-e6d08ae4789d.png)

**Comment by docwilmot** [comment]
Do this! Do it now!

**Comment by wesruv** [comment]
Getting some pro designer feedback... gimme the weekend before anyone hops on this.

**Comment by jenlampton** [comment]
Okay :)

- -
Please excuse brevity and typos. Message typed with my thumbs from an Open
Source OS.

**Comment by wesruv** [comment]
This design will cost $15 for the background texture in the hero, but I think it's worth it 😄 
![image](https://cloud.githubusercontent.com/assets/5607236/20247456/9839cfc8-a99a-11e6-83ee-6f36cdbf4070.png)

A few notes:
* Was staring at the nav, feel like it should reflect the things that are important to us, and thought pulling Community out of Handbook might be a good way to do that
* Changed Spin Up on Pantheon to "Try Out Backdrop", was thinking this could be a page with instructions on setting up Pantheon and/or Kalabox, and if we get any other reasonably quick/easy methods of spinning up a Backdrop instance we can list it there.
* Feel like Download Backdrop should go to a "Getting Started" page that lists out some tips for setting up that also starts the download.

**Comment by wesruv** [comment]
Going to try and get some crits for the Lullabot Design team, see if we can do even better

**Comment by serundeputy** [comment]
First of all the direction this is headed is awesmome.  Thanks for all your hardwork @wesruv !

I just want to point out with regard to this bullet:

> * Changed Spin Up on Pantheon to "Try Out Backdrop", was thinking this could be a page with instructions on setting up Pantheon and/or Kalabox, and if we get any other reasonably 

There is an open discussion about that button and crediting Pantheon for there support for free "try it out" infrastructure to Backdrop CMS devs, testers and, first time demos here: https://github.com/backdrop-ops/backdropcms.org/issues/192

So, I can see if you are going to link off to a page that has more options than just Pantheon like kbox, other hosts, general installation etc, but that makes me think it should be a link under the demo button for learn more ways to demo, try, dev with Backdrop CMS and the button should remain a direct way to quickly (one click from prevalent spot on homepage) spin up a Backdrop instance on Pantheon.

Not claiming to be the end all voice here, but that is my opinion and any direction we take I'd like to make sure https://github.com/backdrop-ops/backdropcms.org/issues/192 discussions are at least considered as a factor.

thanks again, looks great!
~Geoff

**Comment by wesruv** [comment]
Gotcha, my thought was that we have a lot we want to get from our homepage and I don't think we have enough room to give the Pantheon spinup enough credit. I think the current UX is a bit confusing.

If you don't have a Pantheon account, you get dumped to this page:
![image](https://cloud.githubusercontent.com/assets/5607236/20247971/4a784162-a9a8-11e6-9d29-78c557e659ad.png)

If you don't know what Pantheon is, or aren't very familiar with it, the likelihood that you'll fill out this form is probably very low. I don't think we've primed the user enough to know why it's worth it to fill out this form. They don't necessarily understand what they're getting, but they know they're giving away a bunch of their personal info and maybe opting into spam.

I think a separate page can give a very brief and glossy explanation of what that is and give them the option of spinning up a local quickly with Kbox, or a web environment on Pantheon. Extra click, yes, but I think it'll be more likely to succeed and provide a better UX.

I think the Download button should lead to instructions for custom host/shared host/custom local install instructions. Or if we have all instructions on the page, it should be last as it's the least simple.

Basically thinking the download button would link to something more like this, than directly to a ZIP file.
![image](https://cloud.githubusercontent.com/assets/5607236/20248010/372bcfb0-a9a9-11e6-9894-c80bcd4c3bfe.png)

**Comment by serundeputy** [comment]
all good points; I'm on board with it and we can thank/recognize Pantheon on that sub page

thanks @wesruv

**Comment by docwilmot** [comment]
I prefer the previous background pattern; the latest I find a bit more intrusive. 

Re Pantheon button, we can do both: have the button mention "Try Backdrop on Pantheon", out of respect, and still make it navigate to a page that explains what that is. Shouldnt hurt. 

At the bottom of that page, "Other ways to experience Backdrop", listing Kalabox, Automated installers on shared hosts, self-install etc.

**Comment by jenlampton** [comment]
I'm also not sure about the swirly background. I don't think it fits with
our style guide, or the general aesthetic of keeping things simple.

- -
Please excuse brevity and typos. Message typed with my thumbs from an Open
Source OS.

On Nov 13, 2016 2:57 PM, "docwilmot" <notifications@github.com> wrote:

I prefer the previous background pattern; the latest I find a bit more
intrusive.

Re Pantheon button, we can do both: have the button mention "Try Backdrop
on Pantheon", out of respect, and still make it navigate to a page that
explains what that is. Shouldnt hurt.

At the bottom of that page, "Other ways to experience Backdrop", listing
Kalabox, Automated installers on shared hosts, self-install etc.

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub
<https://github.com/backdrop-ops/backdropcms.org/issues/301#issuecomment-260219472>,
or mute the thread

**Comment by jenlampton** [comment]
I'd also like to check in with @dariusgraza since he created the styleguide
(and brand) and see what he thinks.

**Comment by klonos** [comment]
> - Was staring at the nav, feel like it should reflect the things that are important to us, and thought pulling Community out of Handbook might be a good way to do that

👍 

> ...but that makes me think it should be a link under the demo button for learn more ways to demo, try, dev with Backdrop CMS and the button should remain a direct way to quickly (one click from prevalent spot on homepage) spin up a Backdrop instance on Pantheon.

Thinking the same.

> If you don't know what Pantheon is, or aren't very familiar with it, the likelihood that you'll fill out this form is probably very low. I don't think we've primed the user enough to know why it's worth it to fill out this form. They don't necessarily understand what they're getting, but they know they're giving away a bunch of their personal info and maybe opting into spam.

I don't have a pantheon account and I always try to put myself in the shoes of newcomers. What you describe here is more or less how I'd feel about it too.

> - Feel like Download Backdrop should go to a "Getting Started" page that lists out some tips for setting up that also starts the download.

My initial thought was to also add that as a link under the download button, but perhaps this might work better after all. Alternatively, we can rename the "Handbook" link (which makes it sound that people will have to do a lot of reading and which in turn might be off-putting) to "Getting Started" or "Intro".

> Basically thinking the download button would link to something more like this, than directly to a ZIP file.

👍 

As a general note about the buttons: I think they'd be better in the hero (or perhaps right underneath it) rather than lower in the home page. Reason is that when people visit our site, these links should be prominent without requiring scrolling to find them.

> I prefer the previous background pattern; the latest I find a bit more intrusive.

>I'm also not sure about the swirly background. I don't think it fits with our style guide, or the general aesthetic of keeping things simple.

Same here.

**Comment by klonos** [comment]
How do you guys feel about main menu subtitles?

![bizvektor-subtitles-menu-items1](https://cloud.githubusercontent.com/assets/2423362/20279685/98a829e6-aafd-11e6-8218-27ded1a0b933.jpg)

**Comment by wesruv** [comment]
> Re Pantheon button, we can do both: have the button mention "Try Backdrop on Pantheon", out of respect, and still make it navigate to a page that explains what that is. Shouldnt hurt.

I think it does. The current homepage is waaaaaay too text heavy, I'm getting feedback on my design that it's too text heavy. The homepage can only do so many things successfully.

If we have a slick page explaining what spinning up a Pantheon environment means, and that it's free (for everyone until they want to launch) and how quick it is and how cool the features are, then we'd be doing something good for them; we could be sending over actual leads.

If we want a special thanks section, or a supported by section, let's do that. But we shouldn't muddy up the UX of this part of our homepage as a hat tip or thank you.

> demo button for learn more ways to demo, try, dev with Backdrop CMS

@klonos @serundeputy is there a current demo button that I'm missing? Or is this referring to the other thread?

> As a general note about the buttons: I think they'd be better in the hero (or perhaps right underneath it) rather than lower in the home page. Reason is that when people visit our site, these links should be prominent without requiring scrolling to find them.

On this kind of site scrolling doesn't usually effect conversion rate until it gets much further down the page. My theory is that we should do a better job convincing them to try it before the hard sell.

We may get fewer downloads, but my hope is that we get more sites and users which is our real goal.

**Comment by wesruv** [comment]
I'll have a few more mock ups soon

**Comment by docwilmot** [comment]
> I think it does

Pretty sure we're saying the same thing. I was suggesting keeping the word "Pantheon" in the button text, but otherwise do what you said.

**Comment by wesruv** [comment]
I'm saying "Try Backdrop on Pantheon" is too much text for the area as well as a misnomer if we have more than one way to try out Backdrop.

**Comment by wesruv** [comment]
Ok, I think I can have my cake and everyone else can also have cake...

![image](https://cloud.githubusercontent.com/assets/5607236/20313645/8d3b448a-ab25-11e6-9c27-12a45b0efef5.png)

Try Out at the top goes to the page we were talking about before

The gray bar links directly to the form (I think that's more than enough prompting for why you'd fill out the form).

Got the download buttons up top and it doesn't look like crap (really had a hard time with that for some reason)

... this is why I like code more 😬

**Comment by wesruv** [comment]
I also changed the nav more...

I think News can live under community, and it's on the homepage.

Added Download, because we want people to do that, that link should go to a 'Getting Started' page that kicks off a download.

**Comment by docwilmot** [comment]
Still like it. :smile: 

Not so sure about the Nav though. Community can also be under Support, and Extensions are also Downloads. And the difference between the Handbook and Support is also unclear. But that could be another discussion.

**Comment by wesruv** [comment]
Agreed... I'm kind of mixing a lot of tasks into this one 😛

**Comment by jenlampton** [comment]
We don't use the word "extensions" in backdrop. Today we are using
"add-ons". Should we open a bikeshed issue to change it?

I don't like "download" in the nav either. The existing button should be
enough, and it's a better cue to users about what will happen when you
click it (there is no download page yet, is there?).

If the button to try backdrop will go to a normal page and not to a place
to try backdrop, we should also make it not look like a button anymore (or
at least, look different from the download button, if they behave
differently).

I'm also not sure we have anything to say about community yet, and I don't
think the design should dictate content or were likely not to get anything
done. But more to the point...

Can we use the nav items we already have in this issue (consolidating is
fine), and work on reworking the nav in the other issue we have about
reworking the nav? We're starting to go way beyond a home page redesign
here :)

Things we need in the nav today:
- handbook
- add-ons
- support
- news

We can change nav links at any time, and getting consensus might be hard.
let's get back to finalizing design :)

Jen

- -
Please excuse brevity and typos. Message typed with my thumbs from an Open
Source OS.

**Comment by wesruv** [comment]
I think it might be best if I explained what I'm thinking on a call this week.

If it's not clear, I'm thinking of where I might like to see the site, not just what we can do right now. If I only focused on the latter I think I'd be doing a disservice.

We can scale down _all of this_ to make the MVP attainable in a reasonable time-frame and shelf anything that's not quite there or polished that isn't necessary for the MVP.

We did the same thing with Basis, and got a great MVP while I was mocking up and thinking about first run UX with "Getting Started" and how the user nav laid out and I think that process went great.

> We don't use the word "extensions" in backdrop. Today we are using "add-ons". Should we open a bikeshed issue to change it?

For some reason I thought it was Extensions, lol, I have no dog in that fight, Add-ons works for me.

> (there is no download page yet, is there?)

Nope, I'm saying we should have one and I'm going to design that and the Try Out Backdrop pages next.

> I'm also not sure we have anything to say about community yet, and I don't think the design should dictate content or were likely not to get anything done.

I'm not trying to add content with that change, I'm thinking we shuffle things around.

**Handbook > About Backdrop** is more to do with describing our community, and less to do about our Handbook.

Although thinking through the goals and UX of the homepage is exactly the right time to suggest content if there's a hole.

I think we are 'content light' where content could be key, I am suggesting a few new pages:
* Download page
* Try it Out page
* Less features listed on the homepage with a better sell (thinking images or short videos, and bullet points for each feature)

> If the button to try backdrop will go to a normal page and not to a place to try backdrop, we should also make it not look like a button anymore (or at least, look different from the download button, if they behave differently).

They are both calls to action, and they should both go to a page.

Download will kick off a download and then should have helpful content/next steps. These might be installation instructions, links to gitter/github issues/other support channels, ways to get involved or engaged with the project.

The Try Out Backdrop should have the quick and  (relatively) easy ways to setup a site quickly. At the top of the list, Pantheon and a short explanation of what it is, second I vote Kalabox as a way to quickly spin up a local environment, then we could have links to more 'custom' installation steps. (e.g. how to setup with MAMP/WAMP, Ubuntu installation instructions, Drupal VM instructions, whatever how to's we can muster.)

**Comment by wesruv** [comment]
![image](https://cloud.githubusercontent.com/assets/5607236/20320103/a8c62058-ab3e-11e6-8f76-12d2fcd3a85d.png)

**Comment by jenlampton** [comment]
Okay. I'm onboard with all of that, except maybe the handbook. I'd argue
that the about backdrop section is exactly what should be in the handbook,
and all the other things are questionable. (But we do have a separate issue
to rework the handbook after dOCs are done)

We could also Rename the handbook If people are confused about what the
heck we meant that to be. Esp coming from what Drupal did with their
"handbook", there is going to be confusion around that word too.

Words are hard :(

- -
Please excuse brevity and typos. Message typed with my thumbs from an Open
Source OS.

**Comment by tomgrandy** [comment]
FWIW, "Handbook" has never had clarity for me.  By definition, the Handbook section does both what it can do _(a book giving information such as facts on a particular subject or instructions for operating a machine)_ but is it one or both?

About Backdrop is all you need to for clarity.

Documentation is where all the other items residing in the current Handbook section belong.

Within Documentation you can have multiple subject areas:

1. System Requirements
2. Installation
2. User Guide
3. Developer Guide
4. Backdrop API

Two cents.

**Comment by docwilmot** [comment]
I suggest a new issue for bikeshedding the Nav. I've already created a new one for killing off the handbook. See https://github.com/backdrop-ops/backdropcms.org/issues/312. Lets leave this issue for the visual redesign.

**Comment by tomgrandy** [comment]
Thanks for the clarification @docwilmot 

Somehow I missed the above.  Agree with #312 - call me a newb.

**Comment by wesruv** [comment]
Made an issue for the Try Out Backdrop CMS page: https://github.com/backdrop-ops/backdropcms.org/issues/314

**Comment by wesruv** [comment]
Latest and greatest

![image](https://cloud.githubusercontent.com/assets/5607236/20412296/d696d868-acf3-11e6-8be7-48c3d097fb47.png)

**Comment by wesruv** [comment]
Download page: https://github.com/backdrop-ops/backdropcms.org/issues/315

**Comment by bd0bd** [comment]
1. I offer NOT to use "Try out Backdrop" wording.
It is better (winning solution) to use single word "Demo".
Buttons color should be contrast - for example, red.

2. I offer to keep the current colors scheme - black (grays) & white. No blue (but I do understand why it is used by @wesruv). Only the buttons should be another color scheme (for example, red).

Thank you!

**Comment by jenlampton** [comment]
> It is better (winning solution) to use single word "Demo".

If you remember we started by using the word "Demo" but we were asked by Pantheon to change it since that tool is not a demo. What you get is your own Backdrop site - one you can launch! If we do come up with a demo service then it may be better to put a link to that on the home page using the word "Demo" (on simplytest.me maybe?) but the pantheon button needs a more accurate word than that.

**Comment by bd0bd** [comment]
@jenlampton, I understand you.

When I say to use "Demo" I mean you have your own demo on your website (not on Pantheon).

Any visitor could click on this button and a new demo site would be created within 15-90 seconds.
No any registration. No any fields to type in.

**Comment by jenlampton** [comment]
exactly, and the whole thing would be erased in a half hour or so.

**Comment by wesruv** [comment]
In real world implementation land, this is what I've got so far (not done though). From our Design meeting last week, sounds like we'll be putting this up on dev when it's a little more polished. Since we're changing the theme to be something that uses Basis/more core functionality it's causing a redesign of some other pages. Trying to keep that manageable so we can get something out for the release.

My screen grab tool is clipping of the right side a bit... but you get the idea.

## Homepage
### Desktop
![image](https://cloud.githubusercontent.com/assets/5607236/21488465/0779bd14-cbae-11e6-9f32-020ff590647f.png)

### Tablet (closed menu)
![image](https://cloud.githubusercontent.com/assets/5607236/21488471/20ab97da-cbae-11e6-842b-a92ca526486b.png)

### Phone (closed menu)
![image](https://cloud.githubusercontent.com/assets/5607236/21488487/67bd8854-cbae-11e6-8fc7-553ab35fc358.png)

### Phone (open menu)
![image](https://cloud.githubusercontent.com/assets/5607236/21488486/5a6859cc-cbae-11e6-8606-34f70a6d0dbe.png)

## Modules page

### Desktop 
![image](https://cloud.githubusercontent.com/assets/5607236/21488500/98f4a38a-cbae-11e6-9a83-16fdaeeac73d.png)


... more coming soon 😄

Not wed to anything yet, let me know if there's feedback!

**Comment by jenlampton** [comment]
Looks good!

- -
Please excuse brevity and typos. Message typed with my thumbs from an Open
Source OS.

**Comment by klonos** [comment]
Yep, looking good. Any specific feedback once we have it up on the dev server.

**Comment by klonos** [comment]
...we are not redesigning only the home page.

**Comment by tomgrandy** [comment]
@wesruv - Is the dev site available to view?

**Comment by wesruv** [comment]
Not yet, there were more layouts than I thought o_o

**Comment by wesruv** [comment]
Time to check it out!
http://borg.wesruv.com/

A few caveats:
* The "Why use Backdrop CMS" section on the homepage is completely unstyled
* Footer is unstyled
* There are some rough spots on mobile on the homepage, e.g. Pantheon bar

I'll get to those things, but I thought it'd be good to get people looking at this and banging on it and figuring out what else needs help.

Make sure to check out pages like:
http://borg.wesruv.com/requirements
http://borg.wesruv.com/modules/
http://borg.wesruv.com/project/mediaelement
http://borg.wesruv.com/news
http://borg.wesruv.com/account/wesruv
http://borg.wesruv.com/search/node/devel

**Comment by docwilmot** [comment]
Like. Very much.

**Comment by jenlampton** [comment]
Great start!  We will need to remove the swirly-pattern for the background of the blue area, and make it less blue before we launch, but this is looking fantastic @wesruv :) There's now a pattern on the gray background too - I hadn't seen those before. Our brand is SIMPLE so the more noise we have in the backgrounds the more off-brand the design becomes. Can we try the gray areas with no pattern, and maybe move the pattern that's there onto the blue area?  (mabe in design files if changing it in code is too much work).

Also, don't forget to copy the theme functions overrides from the old theme to the new. I notice we lost some of the overrides (like the font-awesome icons in the footer).

**Comment by Dinornis** [comment]
Looks great so far, well done.
Are you aware of how some of the project descriptions are rendered?
This one for example: http://borg.wesruv.com/project/feeds

**Comment by jenlampton** [comment]
@wesruv this looks fantastic. I had no idea how much you were changing. I am so grateful for the time you have put into this, and feel badly that I haven't given this project the attention it deserves! You are a rockstar, thank you :)

**Comment by Dinornis** [comment]
At certain display port widths the login link gets covered with the search bar in Firefox and in Safari the search bar gets chopped off.

![navbarsearchfirefox](https://cloud.githubusercontent.com/assets/24782789/21785389/2bc84722-d724-11e6-8b6e-a8d4b6524122.png)
**^^^ Firefox ^^^**

![navbarsearchsafari](https://cloud.githubusercontent.com/assets/24782789/21785388/2bc1ce92-d724-11e6-90a6-b1a935fb1378.png)
**^^^ Safari ^^^**

**Comment by Dinornis** [comment]
The Pantheon block also has some responsive issues around the 800 width. Sorry 8-(

**Comment by Graham-72** [comment]
This is such excellent work that it seems ungracious to make a critical comment, but when I view it on my conventional PC the page headings and sub headings seem too large a font-size, also the concentric circles in the background are adding to my eyestrain. But thanks @wesruv for all your hard work.

**Comment by jenlampton** [comment]
I'm doing a very thorough review, and there are SO MANY places that had specific styling before, that are either completely unstyled now, or look broken.  @wesruv can you please copy in the components.css file from the old theme? We can go through each component's styles and update as necessary, but we can't have everything back to square zero like this. There's just so much work that still needs to be done (that was already done).

**Comment by klonos** [comment]
A few issues with the hamburger menu on narrow(er) screens...

- Too wide, so the menu is shifted to the left, outside the viewport:

  ![backdropcms org-redesign-hamburger_menu-hidden_off_to_the_left_of_the_screen](https://cloud.githubusercontent.com/assets/2423362/21984402/d2b82398-dc49-11e6-86a7-f540a9507270.png)

- The background of the inactive items seems to be transparent and when the menu height exceeds the height of the header, you get funny results:

  ![backdropcms org-redesign-hamburger_menu-inactive_items_transparent_bg_should_be_same_color_as_header_bg](https://cloud.githubusercontent.com/assets/2423362/21984474/34b0b664-dc4a-11e6-9a49-16280d369b09.png)

  ...perhaps the background color of the menu items of menus placed in the header should match the background color of the header section?

**Comment by quicksketch** [comment]
As usual we covered the redesign in video-form during the fort-nightly design call: https://www.youtube.com/watch?v=3_HFj_tuau8

The relevant portion of our discussion:

- Lets focus on implementation and technical issues in this issue here (as we have so far recently).
- We have a designer (Nica) who will be providing guidance on restyling the hero and key homepage elements.
- Other branding changes in the current implementation (e.g. footer, fonts, bullet lists, etc) are mostly unintentional as side-effects of being a new theme instead of using the existing CSS.

We could use developers doing some work to start iterating on the current branch of work to copy over existing styles into the new theme (including the footer, fonts, handbook styles, etc.)

Anything we don't have explicit styling on, we should raise as needing design help and take that to Nica.

**Comment by jenlampton** [comment]
Here's the latest/final design from Nica for the home page: 
![home](https://cloud.githubusercontent.com/assets/397895/22567851/6185c298-e946-11e6-9f1f-ed0f50a8bef6.png)

I'm working to try to get us from where we are over the finish line today. More updates later.

**Comment by docwilmot** [comment]
Wow its a bit purple blue green!

**Comment by klonos** [comment]
As I mentioned during the design meeting today (or was it the dev meeting?), I really like how the new colorful hero looks around black color and white text. Not so sure about interchanging black and white regions as people scroll down though. White text over black background might be ok for menu items and some small sections, but not that sure about the readability of big portions of white text on black background(?).

**Comment by klonos** [comment]
@docwilmot it's [teal purple and pink](https://www.google.com.au/search?q=teal+purple+and+pink+hair&tbm=isch) and I don't know about the US or other parts of the world, but it seems to be a thing here is Australia 😄 👍

**Comment by wesruv** [comment]
I love it!

**Comment by tomgrandy** [comment]
Looks great!

**Comment by serundeputy** [comment]
i think we can close this as this design was launched some time ago.
re-open if necessary.
